src/aml_service/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const yotiRequest = require('../yoti_request');
+const AmlResultClass = require('../yoti_request/aml.result').AmlResult;
+const Payload = require('../yoti_request/payload').Payload;
+
+const AmlResultObj = function(rawResult){
+  this.onPepList = rawResult['on_pep_list'];
+  this.onFraudList = rawResult['on_fraud_list'];
+  this.onWatchList = rawResult['on_watch_list'];
+}
+
+AmlResultObj.prototype = {
+  /**
+   * Check if user is a politically exposed person.
+   *
+   * @returns {exports.AmlResult.isOnPepList}
+   */
+  isOnPepList() {
+    return this.onPepList;
+  },
+
+  /**
+   * Check if user is on a fraud list.
+   *
+   * @returns {exports.AmlResult.isOnFraudList}
+   */
+  isOnFraudList () {
+    return this.onFraudList;
+  },
+
+  /**
+   * Check if user is on a watch list.
+   *
+   * @returns {exports.AmlResult.isOnWatchList}
+   */
+  isOnWatchList () {
+    return this.onWatchList;
+  },
+
+  /**
+   * Get result data.
+   *
+   * @returns {{}}
+   */
+  getData () {
+    return {
+      'on_pep_list': this.isOnPepList(),
+      'on_fraud_list': this.isOnFraudList(),
+      'on_watch_list': this.isOnWatchList()
+    };
+  },
+
+  toString () {
+    return JSON.stringify(this.getData());
+  }
+}
+
+exports.performAmlCheck = (amlProfile, pem, appId) => {
+  let endpoint = '/aml-check';
+  let httpMethod = 'POST';
+
+  if (!amlProfile) {
+    throw new Error('Error - AmlProfile should be an object of type Entity/AmlProfile');
+  }
+
+  let PayloadObj = new Payload(amlProfile.getData());
+
+  return new Promise((resolve, reject) => {
+    yotiRequest.makeRequest(httpMethod, endpoint, pem, appId, PayloadObj)
+        .then(response => {
+          try {
+            // This will throw an error if the error message is included in the response.
+            AmlResultClass.checkAmlError(response.getParsedResponse());
```

Comment:
Could probably factor out `response.getParsedResponse()`

Reply:
I am not sure why this should be refactored out.

Reply:
Ok I see what you mean, put it in a var instead.

Reply:
ES6 is already in the current version of this package which is already live. So i think that should be fine. 
Also regarding the signed-request, I move it and other functions inside `common` with the idea of sharing them across this package, so if in the future there is any common module that includes it, it should be easy to replace it with that.